platform_main_feature.views_default.inc

```diff
@@ -270,15 +270,15 @@ function platform_main_feature_views_default_views() {
   $handler->display->display_options['pager']['options']['items_per_page'] = '10';
   $handler->display->display_options['style_plugin'] = 'default';
   $handler->display->display_options['row_plugin'] = 'fields';
-  /* Field: Islandora Solr: lcp_mods_facet_date_mdt */
-  $handler->display->display_options['fields']['lcp_mods_facet_date_mdt']['id'] = 'lcp_mods_facet_date_mdt';
-  $handler->display->display_options['fields']['lcp_mods_facet_date_mdt']['table'] = 'islandora_solr';
-  $handler->display->display_options['fields']['lcp_mods_facet_date_mdt']['field'] = 'lcp_mods_facet_date_mdt';
-  $handler->display->display_options['fields']['lcp_mods_facet_date_mdt']['label'] = '';
-  $handler->display->display_options['fields']['lcp_mods_facet_date_mdt']['element_label_colon'] = FALSE;
-  $handler->display->display_options['fields']['lcp_mods_facet_date_mdt']['link_to_object'] = 0;
-  $handler->display->display_options['fields']['lcp_mods_facet_date_mdt']['date_format'] = 'custom';
-  $handler->display->display_options['fields']['lcp_mods_facet_date_mdt']['custom_date_format'] = 'Y-m-d';
+  /* Field: Islandora Solr: lcp_marc_facet_date_dt */
```

Comment:
Guess I was wrong there is marc date being populated (though this object has no marc datastream):

![screen shot 2016-11-25 at 12 17 40](https://cloud.githubusercontent.com/assets/487373/20625144/43f3590c-b309-11e6-81c4-a91e084b2ad3.png)